guillotina/configure/__init__.py

```diff
@@ -447,6 +447,15 @@ def __call__(self, func):
                 )
             # create new class with customizations
             klass = type(func.__name__, (func,), dict(func.__dict__))
+            klass.config = self.config
+            __parameters__ = klass.config.get('parameters', {})
+            if klass.config.get('validate', False):
```

Comment:
Get the schema here and validate it is correct before moving on.

Reply:
yep